kjourney/src/app/app.component.html

```diff
@@ -4,22 +4,161 @@
-  </header>
+    <header></header>
 
-  <a routerLink="home" routerLinkActive="active">Home</a> <!-- TODO: Replace with navbar -->
-  <a routerLink="login">Login</a>
+    <div class="container">
+      <div style="text-align: center;">
+        <img src="assets/newLogo.png" alt="kJourney Logo" display="inline-block" width="25%">
+      </div>
 
-  <router-outlet></router-outlet>
+      <div class="full" style="width:100%">
+        <div class="left" style="width:50%; float:left;">
+          <div class="navbar" style="padding-top:3%">
+            <a routerLink="home" routerLinkActive="active">Home</a>
+            <a routerLink="login">Login</a>
+            <div class="dropdown">
+              <button class="dropbtn">Bands 
+      <i class="fa fa-caret-down"></i>
+              </button>
+              <div class="dropdown-content">
+                <a routerLink="bands">All</a>
+                <a href="#">Executive</a>
+                <a href="#">Leadership Community</a>
+                <a href="#">Principle</a>
+                <a href="#">Manager</a>
+                <a href="#">Consultant</a>
+                <a href="#">Senior Associate</a>
+                <a href="#">Associate</a>
+                <a href="#">Trainee</a>
+                <a href="#">Apprentice</a>
+              </div>
+            </div>
+            <div class="dropdown">
+              <button class="dropbtn">Capabilities & Job Family
+      <i class="fa fa-caret-down"></i>
+              </button>
+              <ul>
+                <div class="dropdown-content">
+                  <li style="list-style:none;">
+                    <a href="#">Sales & Marketing</a>
+                  </li>
+                  <li class="dropdown">
+                  <li class="dropdown-submenu">
+                    <ul class="dropdown-content">
+                      <li style="list-style:none;">
+                        <a href="#">Business Development</a>
+                      </li>
+                      <li style="list-style:none;">
+                        <a href="#">Account Management</a>
+                      </li>
+                    </ul>
+                  </li>
+                  <a href="#">Technical</a>
+                  <a href="#">Consulting</a>
+                  <a href="#">Experience Design</a>
+                  <a href="#">Management</a>
+                  <a href="#">Central Services Teams</a>
+                </div>
+              </ul>
+            </div>
 
-  <footer>
-    <p>© Copyright 2019</p>
-  </footer>
+          </div>
+        </div>
+        <div class="right" style="width:50%; float:right;">
+          <div class="search-bar" style="text-align: right;" display="inline-block;">
+            <input class=”text” type="text" placeholder="Filter..."/>
+            <style></style>
+          </div>
+        </div>
 
+        <router-outlet></router-outlet>
+      </div>
+    </div>
   </body>
 </html>
+
+<script>
+$(document).ready(function(){
```

Comment:
This is jquery code. Is jquery installed by npm, or will this need to be downloaded separately? Where is it imported?

Reply:
inserted jquery script, doesnt affect functionality for this sprint